src/tests/intg/sss_netgroup_thread_test.c

```diff
@@ -0,0 +1,60 @@
+#define _GNU_SOURCE /* for pthread_yield */
```

Comment:
This is a little bit confusing because `pthread_yield()` isn't used in the test.